core/spec.md

```diff
@@ -3691,7 +3693,7 @@ The abstract processing logic would be:
 
 The format of `EXPRESSION` is:
 
-``` text
+```
```

Comment:
I think @clemensv needed some tag to appear on these for his tooling, which is why I picked `text` as it was generic.

Reply:
Ok, I added this for consistency reason - so that some markdown code block renderers don't start *guessing*